mmv1/products/accesscontextmanager/ServicePerimeter.yaml

```diff
@@ -313,6 +313,14 @@ properties:
                   is_set: true
                   item_type:
                     type: String
+                - name: 'roles'
```

Comment:
we're getting missing tests due to a create test not being added in the examples list: refer to https://googlecloudplatform.github.io/magic-modules/test/test/#add-a-create-test

Reply:
We can not use the examples to generate tests because those tests will run in parallel and all of our tests depend on creating a resource that can only be created once at a time. We are looking to refactor our tests in the future but for now this is what we have. I have added this field to the existing tests for all of our resources.